src/scripts/main.js

```diff
@@ -18,10 +18,32 @@ const food = {
   },
 };
 
+const divTree = document.createElement('div');
```

Comment:
Consider appending the newly created 'divTree' element to the document body before assigning it an ID. This ensures that the element is fully configured before being added to the DOM, although it may not cause functional issues in this specific case.